packages/mdc-shape/util.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export function getElevationFromDOM(el) {
+  let elevation = 0;
+  let mdcShapeElevation =
+    getComputedStyle(el).getPropertyValue('--mdc-shape-elevation');
+
+  if (!mdcShapeElevation) {
+    mdcShapeElevation = el.style.getPropertyValue('--mdc-shape-elevation');
+  }
+
+  if (mdcShapeElevation) {
+    elevation = parseInt(mdcShapeElevation);
+  }
+
+  return elevation;
```

Comment:
This is going to silently return 0 in the case where elevation can't be computed by these means. Is that what we want?